### PR TITLE
Use certificates for postgres connection

### PR DESCRIPTION
**Pull request by asmega**
- Production postgres connection now uses AWS RDS certificates
- Also now running circleci tests through docker via docker compose

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -49,6 +51,21 @@ push: login
...
+serve: stop
+	$(DOCKER_COMPOSE) build
+	$(DOCKER_COMPOSE) up -d db
+	./scripts/wait_for_db.sh db postgres
+	$(DOCKER_COMPOSE) up -d app
+
+stop:
+	$(DOCKER_COMPOSE) down -v
+
+spec: stop
```
[the remark this answers is not in the document]
Comment (asmega) [reply Makefile]: its `spec` over here https://github.com/ministryofjustice/fb-submitter/blob/master/Makefile#L31 but happy for the all to be `test`

**Comment by asmega** [comment]
> When running locally, the test suite works well but `make serve` only starts the `db` and not the app. Is this intentional?

So it looks like its booting in production with the db connection string which doesn't work in docker. I will fix

File: Makefile (hunk 1 above)
[the remark this answers is not in the document]
Comment (asmega) [reply Makefile]: i've changed this to `test`

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+      dockerfile: ./Dockerfile
+    environment:
+      DATABASE_URL: "postgres://postgres:password@db/submitter_local"
+      RAILS_ENV: development
+      RAILS_SERVE_STATIC_FILES: "false"
+      MAX_IAT_SKEW_SECONDS: "60"
+      RAILS_LOG_TO_STDOUT: "true"
+      SECRET_KEY_BASE: 'secret_key_base'
+    links:
+      - db
```
Comment (tommotaylor) [inline docker-compose.yml]:
```suggestion
      - db
    ports:
      - "3000:3000"

```

Comment (tommotaylor) [inline docker-compose.yml]:
```suggestion
      - db
    ports:
      - "3000:3000"

```

Comment (tommotaylor) [reply docker-compose.yml]: Allows for instance `http://localhost:3000/health`

Not sure how this would effect production though

Comment (asmega) [reply docker-compose.yml]: it won't affect production and docker compose is not used

**Comment by tommotaylor** [comment]
Nice! Looking forward to all of our apps having a `make test` and a `make serve`, such a nice improvement.

**Pull request merged**